projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-block-editor-assets.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/**
+ * Retrieve resources (styles and scripts) loaded by the block editor.
+ *
+ * @package automattic/jetpack
+ */
+
+declare( strict_types = 1 );
+
+if ( ! class_exists( 'WPCOM_REST_API_V2_Endpoint_Block_Editor_Assets_Controller' ) ) {
```

Comment:
Should this also be updated to match the class? I am not sure this is needed since this file should only be loaded once.

Reply:
Good catch. I removed this in https://github.com/Automattic/jetpack/pull/42314/commits/2a796b3c4cee81229992f244e7c40901eeb8228c.